Makefile

```diff
@@ -14,7 +14,7 @@ else
 	export GC_FLAGS = "all=-trimpath=${PWD}"
 endif
 
-export LD_FLAGS = github.com/pachyderm/pachyderm/src/client/version.AdditionalVersion=$(VERSION_ADDITIONAL)
+export LD_FLAGS="-X github.com/pachyderm/pachyderm/src/client/version.AdditionalVersion=$(VERSION_ADDITIONAL)"
```

Comment:
The problem with -X here is that it will fail the pachctl release (goreleaser/pachctl.yaml). Hence the need to remove it from the Makefile.

Reply:
In that case, we should export an environment variable for the key/value pair without the `-X` (which is not `LD_FLAGS`), because this is an improper format for `LD_FLAGS`.  I'll update this.